### DISCUSSION TITLE
Regarding Usage

### DISCUSSION THREAD

**Discussion by karthikeyanete** [yume-chan/ya-webadb#529] [Q&A]
I have read the description 

 (It started because I want to try the [WebUSB](https://developer.mozilla.org/en-US/docs/Web/API/USB) API, and because I have an Android phone. It's not production-ready, and I don't recommend normal users to try it. If you have any questions or suggestions, please file an issue at [here](https://github.com/yume-chan/ya-webadb/issues).

Similar to this, I have a Scenario 
     Can I use this ? 
     Is there any documentation for this ?

**Comment by yume-chan** [comment]
You can use it. There are documentations in each packages' README, or you can check the demo project for usage.

**Reply by karthikeyanete** [reply]
okay got it

**Comment by karthikeyanete** [comment]
Any idea on this ?  when trying the demo site 

Error: Failed to execute 'claimInterface' on 'USBDevice': Unable to claim interface.

**Reply by yume-chan** [reply]
https://github.com/yume-chan/ya-webadb/discussions/520#discussioncomment-5463841

**Comment by karthikeyanete** [comment]
Tried all the possibilities with different android devices but, still unable to claim interface. Is this is known issue ?

**Reply by yume-chan** [reply]
No, it works for everyone after enabling developer mode on Android and kill Google ADB (and closing all programs that uses ADB like Android Studio, Visual Studio, Scrcpy, Godot, etc.)  on computer
